rick-morty-app-client/src/pages/404/NotFoundPage.tsx

```diff
@@ -1,10 +1,15 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { Container, Button } from "react-bootstrap";
 import { useNavigate } from "react-router-dom";
 
 const NotFoundPage: React.FC = () => {
   const navigate = useNavigate();
 
+  useEffect(() => {
+    const timer = setTimeout(() => navigate("/"), 5000);
```

Comment:
5000 should be const not magic number

Reply:
https://mmcschool.pl/frontend/magic-numbers-czyli-bardzo-zle-praktyki-w-naszym-kodzie/8310/